src/renderer/components/DownloadsDropdown/Update/UpdateAvailable.tsx

```diff
@@ -67,6 +67,16 @@ const UpdateAvailable = ({item, appName}: OwnProps) => {
                         />
                     </LoadingWrapper>
                 </button>
+                <a
+                    className='DownloadsDropdown__Update__Details__Changelog'
+                    onClick={() => window.desktop.openChangelogLink()}
+                    href='#'
+                >
+                    <FormattedMessage
+                        id='renderer.downloadsDropdown.Update.ViewChangelog'
+                        defaultMessage='View Changelog'
+                    />
+                </a>
```

Comment:
Just curious, do we know what is the version it is available for download? Wondering if should mention it as well.